x/feedist/handler_test.go

```diff
@@ -509,6 +464,21 @@ func TestDistribute(t *testing.T) {
 				{dst: weave.Address("address-1"), amount: x.NewCoin(0, x.FracUnit/3, "BTC")},
 				{dst: weave.Address("address-2"), amount: x.NewCoin(0, x.FracUnit/3*2, "BTC")},
 			},
+		}, "whole split into fractions": {
+			recipients: []*Recipient{
+				{Address: weave.Address("address-1"), Weight: 1},
+				{Address: weave.Address("address-2"), Weight: 2},
+			},
+			ctrl: &testController{
+				balance: x.Coins{coinp(2, 0, "BTC")},
+			}, // 2
```

Comment:
What does `// 2` stand for? :thinking: :smile_cat:

Reply:
💕